lib/ui/compositing/scene_host.cc

```diff
@@ -9,8 +9,11 @@
 #include "third_party/tonic/dart_binding_macros.h"
 #include "third_party/tonic/dart_library_natives.h"
 
-#ifdef OS_FUCHSIA
-#include "dart-pkg/zircon/sdk_ext/handle.h"
+#if defined(OS_FUCHSIA)
```

Comment:
I thought this TU was Fuchsia only, can we update the build rules to compile this just on Fuchsia and remove the preprocessor guards?

Reply:
OK, SGTM.

What happens if someone tries to use a SceneHost on another platform?  Right now they can create one, but it will be useless.  After this change, I think even creating one will produce an error because the native hook is unimplemented.  The documentation *does* say "Fuchsia-only" for this class though.  Is all of this OK?